app/build.gradle

```diff
@@ -326,97 +332,6 @@ android {
 }
 
 /**
- * Setup tasks to inject google-services.json file with package name matching
- * the build variant.
-
- * NOTE: My Groovy knowledge is abysmal and Gradle plugins do black magic,
- * hence, the need for an *_APP_ID var for each build variant :(
- */
-afterEvaluate {
-    // Hack to get assets to show up in robolectric tests; try to eventually remove this
-    preCommcareDebugUnitTestBuild.dependsOn mergeCommcareDebugAssets
-
-    processQabetaDebugGoogleServices.dependsOn switchGoogleServicesQabeta
-    processQabetaReleaseGoogleServices.dependsOn switchGoogleServicesQabeta
-    processStandaloneDebugGoogleServices.dependsOn switchGoogleServicesStandalone
-    processStandaloneReleaseGoogleServices.dependsOn switchGoogleServicesStandalone
-    processLtsDebugGoogleServices.dependsOn switchGoogleServicesLts
-    processLtsReleaseGoogleServices.dependsOn switchGoogleServicesLts
-    processCommcareDebugGoogleServices.dependsOn switchGoogleServicesCommcare
-    processCommcareReleaseGoogleServices.dependsOn switchGoogleServicesCommcare
-}
-
-task switchGoogleServicesCommcare(type: Copy) {
-    description = 'Makes build use normal google-services.json file'
-    println project.ext.COMMCARE_APP_ID
-
-    /**
-     * This is an awkward workaround for a Windows build error that occurs when you try
-     * to copy something into the root directory (containing the .gradle folder). Don't
-     * change it without testing on a windows machine
```

Comment:
Confirming that this won't be an issue, the entire process that made this necessary has been removed.